BioArchLinux/diamond/PKGBUILD

```diff
@@ -4,35 +4,32 @@
 
 pkgname=diamond
 pkgver=2.1.0
-pkgrel=1
-pkgdesc="DIAMOND is a sequence aligner for protein and translated DNA searches, designed for high performance analysis of big sequence data https://doi.org/10.1038/s41592-021-01101-x"
+pkgrel=2
+pkgdesc="High performance sequence aligner for protein and translated DNA searches with big sequence data. https://doi.org/10.1038/s41592-021-01101-x"
```

Comment:
avoids `namcap` warning *diamond W: Description should not contain the package name*